Tests/configure_and_test_integration_instances.py

```diff
@@ -743,120 +791,161 @@ def set_docker_hardening_for_build(client, prints_manager):
-    all_module_instances = []
+    Returns:
+        (lst): The server url list to connect to
+    """
 
-    filtered_tests, filter_configured, run_all_tests = extract_filtered_tests(is_nightly=options.is_nightly)
-    tests_for_iteration = tests
-    if run_all_tests:
-        # skip test button testing
-        skipped_instance_test_message = 'Not running instance tests when {} is turned on'.format(RUN_ALL_TESTS_FORMAT)
-        prints_manager.add_print_job(skipped_instance_test_message, print_warning, 0)
-        tests_for_iteration = []
-    elif filter_configured and filtered_tests:
-        tests_for_iteration = [test for test in tests if test.get('playbookID', '') in filtered_tests]
-
-    tests_for_iteration = filter_tests_with_incompatible_version(tests_for_iteration, server_numeric_version,
-                                                                 prints_manager)
-    prints_manager.execute_thread_prints(0)
+    instances_dns = [env.get('InstanceDNS') for env in env_results if ami_env in env.get('Role', '')]
 
-    # get a list of brand new integrations that way we filter them out to only configure instances
-    # after updating content
-    new_integrations_files, modified_integrations_files = get_new_and_modified_integration_files(git_sha1)
-    new_integrations_names, modified_integrations_names = [], []
+    server_urls = []
+    for dns in instances_dns:
+        server_url = dns if not dns or dns.startswith('http') else f'https://{dns}'
+        server_urls.append(server_url)
+    return server_urls
 
-    installed_content_packs_successfully = True
 
-    if LooseVersion(server_numeric_version) >= LooseVersion('6.0.0'):
-        # sleep for one minute before starting to search and install packs to ensure bucket is ready
-        prints_manager.add_print_job('Sleeping for 1 minute...', print_warning, 0)
+def get_json_file(path):
+    with open(path, 'r') as json_file:
+        return json.loads(json_file.read())
+
+
+def configure_servers_and_restart(build, prints_manager):
+    if LooseVersion(build.server_numeric_version) >= LooseVersion('5.5.0'):
+        for server in build.servers:
+            client = demisto_client.configure(base_url=server, username=build.username, password=build.password,
+                                              verify_ssl=False)
+            set_docker_hardening_for_build(client, prints_manager)
+            if LooseVersion(build.server_numeric_version) >= LooseVersion('6.0.0'):
+                set_marketplace_gcp_bucket_for_build(client, prints_manager, build.branch_name, build.ci_build_number, build.is_nightly)
+
+            try:
+                print('Restarting servers to apply server config ...')
+                local_command = 'pgrep -f demisto > xargs kill -9 && server && go mod tidy && go mod vendor && go install -mod=vendor && $GOPATH/bin/server -public ~/dev/demisto/web-client/dist/ -conf conf_for_systemtests -insecure -loglevel debug'
```

Comment:
Oi lo. can you add short comment here that explains this line?

Reply:
copy from `.demisto_bashrc`

Reply:
Leave it 😃